salt/modules/network.py

```diff
@@ -570,9 +570,33 @@ def _ip_route_linux():
 
         # need to fake similar output to that provided by netstat
         # to maintain output format
+        if comps[0] == "unicast":
+            continue
+
+        if comps[0] == "broadcast":
+            continue
+
+        if comps[0] == "throw":
+            continue
+
         if comps[0] == "unreachable":
             continue
 
+        if comps[0] == "prohibit":
+            continue
+
+        if comps[0] == "blackhole":
+            continue
+
+        if comps[0] == "nat":
+            continue
+
+        if comps[0] == "anycast":
+            continue
+
+        if comps[0] == "multicast":
+            continue
+
```

Comment:
**suggestion**

If comps[0] in ('anycast', 'multicast', ....):
    continue

Would that clean things up a bit?

Reply:
done

Reply:
:rocket: nice!

Reply:
(and, that's why we review each others' code, to help catch those things :slightly_smiling_face: )